src/components/sidebar-progess.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <v-stepper class="sidebar-progress" :value="currentStep">
+      <div class="sidebar-progress__steps">
+        <template v-for="(item, index) in items">
+          <v-stepper-step
+            :key="index"
+            class="px-4"
+            :step="index + 1"
+            :complete="item.complete"
+          >{{ item.text }}</v-stepper-step>
+          <v-divider class="px-6" :key="index" v-if="index !== items.length - 1"></v-divider>
+        </template>
+      </div>
+    </v-stepper>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+
+export default {
+  computed: {
+    ...mapState({
+      selections: state => state.selections.selections
+    }),
+    currentStep() {
+      return this.$route.meta.step;
+    },
+    items() {
+      return [
+        {
+          text: 'Select',
+          completed: this.selections.length
+        },
+        {
+          text: 'Calculate',
+          completed: this.$route.name === 'calculate'
+        }
+      ];
+    }
```

Comment:
I'm half okay with this, but in a larger application I wouldn't be happy about creating this super-tight coupling between steps, route meta, and this item data inside the component. Rather an item would be an object literal with a number of properties that then gets imported and passed around to the relevant components/routes, which pull the data they need off that object. Up to you if you want to go for this refactor, do it later, or leave it as is alltogether ☺️